axes/decorators.py

```diff
@@ -412,15 +494,14 @@ def check_request(request, login_unsuccessful):
                 )
                 attempt.http_accept = \
                     request.META.get('HTTP_ACCEPT', '<unknown>')[:1025]
-                attempt.path_info = \
-                    request.META.get('PATH_INFO', '<unknown>')[:255]
+                attempt.path_info = path_info,path_info
```

Comment:
I haven't fixed, I just noticed it was there. Please make a PR fixing it. Thanks.